docs/Data/File_Formats/MAF_Format.md

```diff
@@ -45,178 +55,33 @@ The process for modifying a protected MAF into a somatic MAF is as follows:
-
-## Impact Categories
-
-### VEP
-
-* __HIGH (H)__: The variant is assumed to have high (disruptive) impact in the protein, probably causing protein truncation, loss of function, or triggering nonsense mediated decay
-* __MODERATE (M)__: A non-disruptive variant that might change protein effectiveness
-* __LOW (L)__: Assumed to be mostly harmless or unlikely to change protein behavior
-* __MODIFIER (MO)__: Usually non-coding variants or variants affecting non-coding genes, where predictions are difficult or there is no evidence of impact
-
-### PolyPhen
-
-* __probably damaging (PR)__: It is with high confidence supposed to affect protein function or structure
-* __possibly damaging (PO)__: It is supposed to affect protein function or structure
-* __benign (BE)__: Most likely lacking any phenotypic effect
-* __unknown (UN)__: When in some rare cases, the lack of data does not allow PolyPhen to make a prediction
-
-### SIFT
-
-* __tolerated__: Not likely to have a phenotypic effect
-* __tolerated_low_confidence__: More likely to have a phenotypic effect than 'tolerated'
-* __deleterious__:  Likely to have a phenotypic effect
-* __deleterious_low_confidence__: Less likely to have a phenotypic effect than 'deleterious'
+### Aliquot-Level MAF Files (Data Release >17.0)
+
+Aliquot-level MAF files, annotated somatic mutations, are produced for each aliquot per variant caller. These files are then run through the Aliquot Ensemble Somatic Variant Merging and Masking workflow. There are a few filters that are applied at this step. The variants must be somatic, the variant size must be ≤ 50 bp, and it must pass the filters for the caller, except for MuSE which passes on filters for Tier 1-4 and the panel of normals. From this workflow two files are produced, aggregated somatic mutation and masked somatic mutation. The aggregated somatic mutation file is the aggregation of all variants from the multiple variant callers for each aliquot with these applied filters. The masked somatic mutation file is the aggregation of all variants from the multiple variant callers for each aliquot, which are then passed through a second filtering process.
+
+#### Aliquot-Level Masked Somatic Mutation File Generation
+
+The process for modifying the aliquot-level MAF files into a masked somatic mutation aliquot-level MAF is as follows:
+
+*  The annotated somatic mutations aliquot-level MAF files, produced from the different callers, are merged into one raw merged aliquot-level MAF file. Then selection for the variants are made based on the following low quality variant filtering and germline masking:
+    1. The variant must occur within at least two of the callers.
+    2. Remaining variants with __FILTER != panel_of_normals__ are __removed__. Note that the `FILTER != panel_of_normals` value is only relevant for the variants generated from the MuTect2 pipeline.
+    3. The __non-TCGA exac allele frequency__ variants (0.001; common\_in\_exac) are __kept__.
```

Comment:
I'm confused, we remove variants with non-TCGA ExAC allele frequency > than the cutoff; however, they can be rescued based on item number 4